README.md

```diff
@@ -100,8 +100,6 @@ Fill in the missing pieces in [this](https://github.com/Azure/kubernetes-keyvaul
     |keyvaultobjectaliases|no|filenames to use when writing the objects|keyvaultobjectnames|
     |keyvaultobjecttypes|yes|types of Key Vault objects: secret, key or cert|""|
     |keyvaultobjectversions|no|versions of Key Vault objects, if not provided, will use latest|""|
-    |resourcegroup|yes|name of resource group containing Key Vault instance|""|
```

Comment:
Let's keep these properties here for now for backward compatibility. Optionally you can update the description to `[required] for version < v0.0.13`. we can consider removing these from docs and examples after v+2.